app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -1,116 +1,92 @@
+The school system in England is split into age groups:
 
-If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teacher status](/what-is-qts) (QTS) which you will need to teach in the majority of schools in
-England.
+### Primary (ages 5 to 11)
+Children usually start primary school at 4 years old and this phase is called Reception. This is the final stage of the Early Years Foundation Stage (EYFS).
 
-You will need a degree to teach all student groups except those in further education, depending on the type of further education you are teaching.
+There are a further 2 phases, known as key stages: 
 
-## Teach disabled pupils and pupils with special educational needs
+* key stage 1 covers years 1 and 2, with children up to 7 years old
+* key stage 2 covers years 3 to 6, with children from 7 to 11 years old
 
-Most disabled pupils and pupils with special educational needs learn in mainstream schools.
-No matter what settings you train and work in – whether it’s mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
+The teaching content is broad, covering the whole range of national curriculum subjects including English, maths, science, as well as art and design, computing, geography, history, music and physical education (PE).
 
-You can [find a course which specialises in special educational needs and disability (SEND)](https://www.find-postgraduate-teacher-training.service.gov.uk/) if you have a particular interest in this area.
+If you like the energy and curiosity of young children, and would be comfortable covering a wide range of subjects, you might prefer teaching in primary schools. 
 
-However, it’s not essential to do a course specialising in SEND. As you train and teach you’ll get the skills you need to work with disabled pupils and pupils with special educational needs. This will enable you to work in a range of settings throughout your career.
+If you are interested in training to be a primary teacher, you could consider options for primary teaching with a subject specialism like maths or English. This will cover all of the subjects but with a focus on your subject specialisation.
 
-### Teach pupils with sensory impairments
+### Secondary (ages 11 to 18)
+Children start secondary school when they’re 11. There are up to 3 stages in secondary school: 
 
-You need [specific qualifications to teach a class of pupils with hearing impairments, vision impairments or multi-sensory impairments](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers).
+* key stage 3 (KS3) includes years 7 to 9, with children aged 11 to 14  
+* key stage 4 (KS4) includes years 10 and 11, with pupils aged 14 to 16
+* key stage 5 (KS5) includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms or colleges
```

Comment:
```suggestion
* key stage 5 (KS5) includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms, or colleges
```

Not sure whether this is stylistically acceptable, to have a comma here (there may be a better solution) - but this read to me as 'secondary schools with colleges' rather than separate entities